--- libwx-perl.orig/build/Wx/XSP/Node.pm
+++ libwx-perl/build/Wx/XSP/Node.pm
@@ -1,8 +1,6 @@
 package Wx::XSP::Node;
 
-=head1 Wx::XSP::Node
-
-Base class for the parser output.
+=head1 Wx::XSP::Node \- Base class for the parser output.
 
 =cut
 
--- libwx-perl.orig/build/Wx/XSP/Parser.pm
+++ libwx-perl/build/Wx/XSP/Parser.pm
@@ -5,7 +5,7 @@
 use IO::Handle;
 use Wx::XSP::Grammar;
 
-=head1 Wx::XSP::Parser METHODS
+=head1 Wx::XSP::Parser \- an XS++ parser
 
 =cut
 
--- libwx-perl.orig/build/Wx/XSP/Typemap.pm
+++ libwx-perl/build/Wx/XSP/Typemap.pm
@@ -1,6 +1,6 @@
 package Wx::XSP::Typemap;
 
-=head1 Wx::XSP::Typemap
+=head1 Wx::XSP::Typemap \- map types
 
 =cut
 
